stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1071,7 +1099,12 @@ func _TFCEagerExecute(_ op: CTFEOp,
                       _ status: CTFStatus) {
   if _RuntimeConfig.printsDebugLog {
     debugLog("Calling _TFCEagerExecute() over: ")
-    TFE_OpPrintDebugString(op)
+    if let value = getenv("TF_CPP_MIN_LOG_LEVEL"),
```

Comment:
This already gets printed only if SWIFT_TENSORFLOW_ENABLE_DEBUG_LOGGING is set? Do you also want to control it with TF_CPP_MIN_LOG_LEVEL?

Reply:
If `SWIFT_TENSORFLOW_ENABLE_DEBUG_LOGGING` is set to `true` but `TF_CPP_MIN_LOG_LEVEL` is not set to `0`, compiler runtime prints `Calling _TFCEagerExecute() over: ` and then nothing. I figured that this behavior is undesirable hence this change that suggests that the user specifies `TF_CPP_MIN_LOG_LEVEL=0`.